### DISCUSSION TITLE
Some merged items to the library doenst work

### DISCUSSION THREAD

**Discussion by skank01** [andrew-codechimp/HA-Battery-Notes#2010] [General]
I asked to merge some new batteries.
Some of them are now recognized, but some of them not.
I answered in the closed pull request

**Comment by andrew-codechimp** [comment]
All the devices you submitted have ben merged, you can see the list [here](https://github.com/andrew-codechimp/HA-Battery-Notes/blob/main/library.md)
I modified some as Model ID is a new feature which is not used much at all yet and I doubted it was duplicated from the model, the model id is an optional match and so it should still be found.
A restart of HA will pull the very latest library down in case yours hasn't be refreshed yet.

**Reply by skank01** [reply]
I again restarted to pull latest library but still, they aint showing.
Did i do something wrong?

For example the aqara one
![image](https://github.com/user-attachments/assets/98bc0e49-6941-45df-9de2-07458a9c697c)


Eufy ones
![image](https://github.com/user-attachments/assets/65dad491-9de9-46e6-b32a-07eee110a491)

![image](https://github.com/user-attachments/assets/0eb7b427-182c-40b4-8e22-9cb094002c3e)


all not showing

**Comment by andrew-codechimp** [comment]
That Aqara one is awkward, I've put a wildcard match on the (LYWSD03MMC-z) part.
The Eufy ones should have a manufacturer of Eufy Security, so I've changed those.
A new library is now available so another HA restart will get it, I'm also looking at a better way to show these for adding to the library, they are a little tricky.

**Comment by skank01** [comment]
Hi, thx, the 2 cameras from eufy are now found.
The agara one is not :( (i have lots of those devices lol)

**Reply by andrew-codechimp** [reply]
Just updated the library again so please try again when you get a chance.

**Comment by skank01** [comment]
yes working now, thx !
